### DISCUSSION TITLE
How to get preview versions?

### DISCUSSION THREAD

**Discussion by bbuchanan** [Megabit/Blazorise#1741] [Q&A] [answered]
This is probably a general question but I've never attempted to get a preview version of any package. I'm using dotnet on the command line and I'd really like to try out the features in 0.9.5-preview6 but don't know how to obtain it. Can anyone kindly point me in the right direction?

Also, is there a release date for 0.9.5 posted?

Thank you.

**Comment by stsrki** [comment, marked as answer]
Hi!

You have a guide in the README section: https://github.com/stsrki/Blazorise#try-preview

**Reply by bbuchanan** [reply]
ah sorry about that. Thanks for your response. Really enjoying getting to know your library. Thank you!
